contracts/PushComm/PushCommV3.sol

```diff
@@ -689,21 +689,33 @@ contract PushCommV3 is Initializable, PushCommStorageV2, IPushCommV3, PausableUp
      */
     // Should be only admin
     // Should only bridge NTT TOKENS FROM COMM TO CORE on ethereum
-    function transferFeePoolToCore(uint256 _amount) external payable onlyPushChannelAdmin {
+    function transferFeePoolToCore(CrossChainRequestTypes.CrossChainFunction functionType, uint256 _amount, uint256 gasLimit) external payable onlyPushChannelAdmin {
```

Comment:
no need of taking functionType in the param, AdminRequest_AddPoolFee can be used directly here in this func

Reply:
done